verification/cbmc/proofs/aws_hash_table_clean_up/aws_hash_table_clean_up_harness.c

```diff
@@ -27,6 +27,7 @@ void aws_hash_table_clean_up_harness() {
     // This would normally raise a warning since the memory was deallocated, so use the pragma.
 #pragma CPROVER check push
 #pragma CPROVER check disable "pointer"
+#pragma CPROVER check disable "pointer-overflow"
```

Comment:
Recently I pushed a bunch of PRs to s2n that formally verify that deallocator functions don't have any memory leak (i.e. they clean up everything that they are supposed to clean up).

I think we should also create a separate GitHub ticket to also do this for c-common at some point. The disable `pragma`s are bad, but after removing them the harness would basically be empty. Instead of having an empty harness, we might want to verify that deallocation is leak-proof.

@feliperodri What do you think?

Reply:
Let's create a GitHub issue for to track this request on a separate PR.

Reply:
Created #805.